src/main/java/at/makubi/maven/plugin/avrohugger/mapping.java

```diff
@@ -0,0 +1,6 @@
+package at.makubi.maven.plugin.avrohugger;
+
+public class mapping {
+    String from;
```

Comment:
I am not sure, if this works or if you need to use POJO with setters and getters.

Reply:
I tested it out myself and this works.